be/src/storage/metadata_util.cpp

```diff
@@ -225,6 +225,22 @@ Status convert_t_schema_to_pb_schema(const TTabletSchema& tablet_schema, uint32_
     }
     schema->set_compress_kind(COMPRESS_LZ4);
 
+    switch (compression_type) {
+    case TCompressionType::LZ4_FRAME:
+    case TCompressionType::LZ4:
```

Comment:
Why not use a map to store the supported compression types and corresponding actions